client/src/components/Istoric.component.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect } from 'react';
+import { FaAngleRight, FaAngleLeft } from 'react-icons/fa';
+
+interface CarouselItem {
+  photo: string;
+  text: string;
+  name: string;
+}
+
+interface CarouselProps {
+  items: CarouselItem[];
+}
+
+const Carousel: React.FC<CarouselProps> = ({ items }) => {
+  const [currentIndex, setCurrentIndex] = useState(0);
+  const [timer, setTimer] = useState<number>(0);
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setTimer((prevTimer) => prevTimer + 1000); 
+    }, 1000); 
```

Comment:
i think is too fast.

Reply:
the timer is not there the timer is set to 7000 in the useEffect

Reply:
that useEffect is just for incrementation of the timer not for setting its max value

Reply:
still is too fast, i can t read all the paragraph without changing.

Reply:
On the old website the duration was about 4-5 sec. Still i can modify the duration if needed.